src/pages/ProductDetail/ProductDetail.scss

```
@@ -1,3 +1,380 @@
-.ProductDetail {
-  font-size: 20px;
+@import '/src/styles/reset.scss';
+
+.productDetail {
+  width: 100%;
+  padding-top: 80px;
+  line-height: 1.2;
+  .orderWrapper {
+    margin: auto;
+    width: 1280px;
+    .leftWrapper {
+      width: 47%;
+      .productImgWrapper {
+        .relationTo {
```

className선택자와 다음 선택자 사이에는 공백을 한 줄 씩 ㄴ넣어주세요!
```suggestion
.productDetail {
  width: 100%;
  padding-top: 80px;
  line-height: 1.2;
  
  .orderWrapper {
    margin: auto;
    width: 1280px;
    
    .leftWrapper {
      width: 47%;
      
      .productImgWrapper {
        .relationTo {
```